app/Interfaces/LabelProvider.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Interfaces;
+
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+interface LabelProvider
```

Comment:
"Provider" for an interface? This is confusing. Why not "LabelInterface"?

Reply:
Good point